### PR TITLE
WIP: Add explore pages

### PR DESCRIPTION
**Pull request by bkeepers**
This adds an explore page at `/apps` that will list all official apps, and adds browsing by topic.

### Topics

I looked through the existing apps and attempted to categorize them. Here are the topics that seemed to make sense with the current plugins. Thoughts?

- administration
- code-review
- community-management
- licensing
- project-management

### TODO

- [ ] Get feedback on list of topics
- [ ] browse hosted apps by topic
- [ ] link to browse [all open source apps](https://github.com/search?q=topic%3Aprobot-plugin&type=Repositories)
- [ ] Add test to ensure all apps are in at least 1 topic and all topics are valid

### CODE REVIEW

File: _apps/snooze.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ author: jbjonesjr
 stars: 3
 installations: 5
 repository: jbjonesjr/probot-snooze
+topics:
+- issues
```
Comment (hiimbex) [inline _apps/snooze.md]: Maybe this should be community-management?

**Comment by hiimbex** [comment]
I think topics are a great idea! ✨ Licensing and Administration are both kind of one off's, but I think they make sense given DCO ad Settings are literally that. So I think they all make sense!

File: _apps/snooze.md (hunk 1 above)
Comment (bkeepers) [inline _apps/snooze.md]: 👍

**Comment by bkeepers** [comment]
Closing this since I don't have plans to work on it anytime soon and there are a million conflicts. Once we get more apps, we'll revisit.

**Pull request closed without merging**